### ISSUE TITLE
Feature request: Fuse linear and Batch Norm layers

### ISSUE THREAD

**Issue by ternaus** [PyTorchLightning/pytorch-lightning#2544]
Neural Networks become more and more popular with tabular data.

Similar to https://pytorch.org/docs/stable/_modules/torch/quantization/fuse_modules.html#fuse_modules

It would be great if you added the way to merge Linear and BatchNorm1d layers.

**Comment by pintonos** [comment]
Any updates on this?
